test/test_import_csv.py

```diff
@@ -0,0 +1,150 @@
+import tempfile
+import unittest
+import app_engine_test_utils
+import import_csv
+from gae.room import ndb_models
+from test import test_models
+
+
+class TestImportCsv(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    app_engine_test_utils.activate_app_engine_testbed_and_clear_cache()
+    test_models.CreateAll()
+
+  def testImportSitesAndCaptains(self):
+
+    # dump test data to a tmp files
+    path_to_sites_data = tempfile.mktemp()
+    with open(path_to_sites_data, 'wb') as io:
+      io.write(TestImportCsv.SITES_DATA)
+    path_to_captains_data = tempfile.mktemp()
+    with open(path_to_captains_data, 'wb') as io:
+      io.write(TestImportCsv.CAPTAINS_DATA)
+
+    import_csv.import_sites(path_to_sites_data, 2442)
+    import_csv.import_captains(path_to_captains_data)
+
+    site = ndb_models.NewSite.query().filter(ndb_models.NewSite.number == "18010SAM").get()  # type: ndb_models.NewSite
+    self.assertTrue(site)
+    site_captains = list(site.sitecaptain_set)
+    self.assertEqual(len(site_captains), 2)
+    for site_captain in site_captains:
+      ctype = site_captain.type
+      captain = site_captain.captain.get()
+      self.assertIn(ctype, ("Construction", "Volunteer"))
+      if ctype == "Construction":
+        self.assertEqual(captain.email, 'mike@nibbi.com')
+      else:
+        self.assertEqual(captain.email, 'bobn@nibbi.com')
+
+
+  SITES_DATA = """\
+Announcement Subject,Announcement Body,Site ID,Budgeted Cost in Campaign,Repair Application: Applicant's Name,Applicant Home Phone,Applicant Mobile Phone,Applicant Work Phone,Recipient's Street Address,Recipient's City,Recipient's Zip Code,Jurisdiction,Sponsor,Repair Application: RRP Test Results,Photos Link
+subject of announcement,"announcement body, has some commas",18001DAL,"$10,000",Man Shu Lau,,(415) 350-7499,,221 South Hill Court,Daly City,94014,Daly City,Woodlawn Foundation & St. Andrews Church (volunteers),"Not tested - Not required to test, assume no lead",https://flic.kr/s/aHsksxU9or
```

Comment:
also here, real data is not appropriate. let's invent some made-up world based on Tiago's imagination.